tests/test_base_api_client.py

```diff
@@ -38,7 +38,7 @@ def _empty_context_manager():
 class TestBaseApiClient(object):
     def _from_httplib_response_mock(self, status, response_data=None):
         response_mock = mock.Mock(
-            status=status, headers={}, spec=['get_redirect_location', 'getheader', 'read', 'reason']
+            status=status, headers={}, spec=['get_redirect_location', 'getheader', 'read', 'reason', 'drain_conn']
```

Comment:
Why is this needed now when it wasn't before?

Reply:
Good question - I spent quite a while trying to figure this out yesterday with no definitive answer. My best guess is that the pinned range for `requests` on this repo allowed an updated version that introduced this extra method (which we weren't previously stubbing out on our mock).

Reply:
Interesting. This might be something pip-tools/Dependabot could help with.

Reply:
A problem for another day though.